plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/editor.scss

```diff
@@ -0,0 +1,27 @@
+.wp-block-woocommerce-add-to-cart-with-options-variation-selector-attribute-options__pills {
+	display: inline-flex;
+	gap: $grid-unit-10;
+	list-style: none;
+	padding: 0;
+	margin: 0;
+	flex-wrap: wrap;
+}
+
+.wp-block-woocommerce-add-to-cart-with-options-variation-selector-attribute-options__pill {
+	border: 1px solid $black;
+	border-radius: $grid-unit-40;
+	padding: 0.25rem 1rem;
+	text-transform: uppercase;
+	font-size: var(--wp--preset--font-size--small);
```

Comment:
Not a big deal, but given that this depends on the theme (I think), I would leave a hard-coded value in `em`  instead. Then in the future we can add support for typography styles if needed.

Reply:
Never mind, I see we are doing the same in many other blocks, so I guess it's safe.